ddtrace/contrib/internal/pytest/_plugin_v2.py

```diff
@@ -275,6 +280,15 @@ def pytest_configure(config: pytest_Config) -> None:
             if _is_pytest_cov_enabled(config):
                 patch_coverage()
 
+            skip_pytest_runtest_protocol = False
+
+            for plugin in ("flaky", "rerunfailures"):
```

Comment:
Maybe worth setting this tuple as an INCOMPATIBLE_PLUGINS constant, so it's easier to maintain?